templateutils/funcs.go

```diff
@@ -198,11 +206,11 @@ func findField(fieldPath []string, v reflect.Value) reflect.Value {
 	return f
 }
 
-func filterField(obj interface{}, field, val string, cmp func(string, string) bool) (retval interface{}) {
+func filterField(fnname string, obj interface{}, field, val string, cmp func(string, string) bool) (retval interface{}) {
```

Comment:
This turned out to be a good catch.  retval was not needed at all!  I've removed the identifier and renamed fnname.